vere/serf.c

```diff
@@ -360,34 +368,6 @@ _serf_sure(u3_noun ovo, u3_noun vir, u3_noun cor)
   u3z(u3A->roc);
   u3A->roc = cor;
 
-  //  single-home
-  //
-  //    XX revise when real keys are supported
-  //    XX dispatch on evt_d, wire, or card tag?
-  //
-  if ( c3__boot == u3h(u3t(ovo)) ) {
-    //  ovo=[%boot *]
-    //  vir=[[wire %init @p] ~]
-    //  fec=[%init @p]
-    //
-    u3_noun fec = u3t(u3h(vir));
-
-    c3_assert( c3__init == u3h(fec) );
-    c3_assert( u3_none == u3A->our );
-
-    u3A->our = u3k(u3t(fec));
-    u3A->fak = ( c3__fake == u3h(u3t(u3t(ovo))) ) ? c3y : c3n;
-
-    {
-      u3_noun nam = u3dc("scot", 'p', u3k(u3A->our));
-      c3_c* nam_c = u3r_string(nam);
-      fprintf(stderr, "boot: ship: %s%s\r\n", nam_c,
-                                      (c3y == u3A->fak) ? " (fake)" : "");
-      free(nam_c);
-      u3z(nam);
-    }
-  }
-
```

Comment:
As mentioned in the PR explanation, `u3A->our` and `u3A->fak` are now set by the `%boot` command, so this single-homing is no longer strictly necessary. The only exception might be during full event replay.

On master, when we replay, we learn who we are by observing the `%init` effect of the `%boot` event, as above. Going forward here, I'd like to persist that metadata somewhere else (perhaps in the event log, as an event 0, since real events start at 1).  But that's not yet implemented. So removing this now is a little aggressive.

Reply:
Boot metadata is now persistently stored in "event 0" .